invite/templates/invite/denied.html

```diff
@@ -3,5 +3,5 @@
 Access Denied
 {% endblock %}
 {% block content %}
-That is an invalid or expired activation code.
+That is an invalid or expired activation code. You may have already registered an account. Try using the "Forgot Password" link on the edit login page to retrieve your credentials via email.
```

Comment:
We could change the last sentence to something like "Try retrieving your credentials via email." with part or all of that sentence linked to the amnesia page.